src/i18n/strings/en_EN.json

```diff
@@ -2210,6 +2210,7 @@
         "sort_by": "Sort by",
         "sort_by_activity": "Activity",
         "sort_by_alphabet": "A-Z",
+        "sort_sections": "Group rooms by type",
```

Comment:
This is subject to change, feel free to suggest a better alternative. As long as we don't have section headers, "Sort rooms by type" might make more sense anyway.